Evaluate-ModelarDB-Compression/main.py

```diff
@@ -342,12 +342,17 @@ def send_sigint_to_process(process):
                 try:
                     test_data_field_column = test_data.column(field_column_name)
                 except:
-                    # Spaces in the name may have been replaced by underscores.
-                    field_column_name_with_space = field_column_name.replace("_", " ")
-                    test_data_field_column = test_data.column(
-                        field_column_name_with_space
-                    )
-
+                    if field_column_name.__contains__(" "):
+                        # Spaces in the name may have been replaced by underscores.
+                        field_column_name_with_space = field_column_name.replace("_", " ")
+                        test_data_field_column = test_data.column(
+                            field_column_name_with_space
+                        )
+                    else: 
+                        # Camel case columns may have been converted to lower case.
```

Comment:
Maybe change to `Upper case columns may have been converted to lower case.` since it can be any kind of upper case, not just camel case.

Reply:
Yes, it makes more sense, fixed :)